notebooks/ch14/bias_modeling.md

```diff
@@ -229,15 +230,28 @@ Cross-validation provides a more accurate method of estimating our model error u
 
 The bias-variance tradeoff allows us to more precisely describe the modeling phenomena that we have seen thus far.
 
-Underfitting is typically caused by too much bias; overfitting is typically caused by too much variance.
+Underfitting is typically caused by too much bias; overfitting is typically caused by too much model variance.
 
-Collecting more data reduces variance. One recent trend is to select a model with low bias and high intrinsic variance (e.g. a neural network) and collect many data points so that the model variance is low enough to make accurate predictions. While effective in practice, collecting enough data for these models tends to require large amounts of time and money.
+Collecting more data reduces variance. For example, the model variance of linear regression goes down by a factor of $ 1
+/n $, where $ n $ is the number of data points. Thus, doubling the dataset halves the model variance, and collecting many data points will cause the variance to approach 0. One rece end is to select a model with low bias and high intrinsic variance (e.g. a neural network) and collect many data points so that the model variance is low enough to make accurate predictions. While effective in practice, collecting enough data for these models tends to require large amounts of time and money.
 
 Collecting more data reduces bias if the model can fit the population process exactly. If the model is inherently incapable of modeling the population (as in the example above), even infinite data cannot get rid of model bias.
 
 Adding a useful feature to the data, such as a quadratic feature when the underlying process is quadratic, reduces bias. Adding a useless feature rarely increases bias.
 
-Adding a feature, whether useful or not, typically increases model variance. In order to increase a model's prediction accuracy, a new feature should decrease bias more than it increases variance.
+Adding a feature, whether useful or not, typically increases model variance. This does so because each new feature addas a a new model parameter, increasing the total number of models. In order to increase a model's prediction accuracy, a new feature should decrease bias more than it increases variance. 
```

Comment:
>increases model variance because each new feature adds a new model parameter, increasing the number of possible models 

Comes from what you wrote.
Doesn't adding an additional parameter create new types of models though?   If your model is $\theta_1 *x$ and you add a quadratic parameter $ x^2$, isn't $\theta_1 *x + theta_2 *x^2$ a new model?

Reply:
Yeah, I know that part came from what I wrote. Sorry about that! I was in a rush when I first wrote the change.

I agree that those are two different models. However, "increasing the total number of models" suggests that there used to be X total models in the universe and now we've increased the total number of models from X to X + 1. In reality, adding a new parameter to a model gives the model more possible combinations of parameters which increases model variance.